ui/react-components/utils/SMSUtil.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash'
+import {isVirtual} from "../components/AddAppointment/AddAppointment.jsx";
+import { 
+    enableAppointmentBookingSMSAlert,
+    messageType,
+    clinicName
+} from "../config";
+import {getPersonAttribute} from "../api/patientApi";
+
+export const isAppointmentSMSEnabled = (appConfig) => {
+    return appConfig.enableAppointmentBookingSMSAlert || enableAppointmentBookingSMSAlert;
+};
+
+const getMessage = (data, message, clinic, recurring = null) => {
```

Comment:
SMSUtils file -- what is its purpose. Is this for generating messages for Appointments? If so, then this should be called: SMSAppointmentUtils.js, or even better, AppointmentSMS.js.

Also, the appointments module frontend has a different code repository, so I feel this code needs to go there. Thoughts? Maybe @binduak can shed some clarity on this.